aries-core/src/js/components/core/index.js

```diff
@@ -1,16 +1,6 @@
-export * from './Accordion';
-export * from './Button';
-export * from './Checkbox';
 export * from './Form';
```

Comment:
Currently, Form is just a pass through. Should we eliminate it from `aries-core` for the time being? I know we have discussed it becoming an `aries-core` component eventually, but I think it is a quarter+ out on the roadmap. For consistency and clarity, should we treat it as the rest of the grommet components until it justifies being moved over?

Reply:
Good comment. I will clean that out and take out any others that don't already have specifications.